RELEASE_NOTES.md

```diff
@@ -1,5 +1,53 @@
 This file holds "in progress" release notes for the current release under development and is intended for consumption by the Chef Documentation team. Please see <https://docs.chef.io/release_notes.html> for the official Chef release notes.
 
+# Chef Infra Client 15.1
+
+## New Resources
+
+### chocolatey_feature
+
+The `chocolatey_feature` resource allows you to enable and disable Chocolatey features. See the [chocolatey_feature documentation](https://docs.chef.io/resource_chocolatey_feauture.html) for full usage information. Thanks [@gep13](https://github.com/gep13) for this new resource.
+
+## Updated Resources
+
+### chocolatey_source
+
+The `chocolatey_source` resource has been updated with new `enable` and `disable` actions as well as `admin_only` and `allow_self_service` properties. Thanks [@gep13](https://github.com/gep13) for this enhancement.
+
+### launchd
+
+The `launchd` resource has been updated with a new `launch_events` property, which allows you to specify higher-level event types to be used as launch-on-demand event sources. Thanks [@chilcote](https://github.com/chilcote) for this enhancement.
+
+### yum_package
+
+The `yum_package` resource's helper for interacting with the yum subsystem has been updated to always close out the rpmdb lock even during failures. This may prevent the rpmdb becoming locked in some rare conditions. Thanks for reporting this issue [@lytao](https://github.com/lytao).
```

Comment:
```suggestion
The `yum_package` resource's helper for interacting with the yum subsystem has been updated to always close out the rpmdb lock, even during failures. This may prevent the rpmdb becoming locked in some rare conditions. Thanks for reporting this issue, [@lytao](https://github.com/lytao).
```